.github/workflows/code_style.yaml

```diff
@@ -8,6 +8,18 @@ on:
       - prerelease_test
   pull_request:
 jobs:
+  perl_checks:
+    name: Check Perl code in tree
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install prerequisites
+        run: sudo apt install perltidy
```

Comment:
Is there a way to update the base image with our requirements instead of running `apt` all the time?

Reply:
Not that I have found.